patches/0001-Patch-Tools-for-sdk-pr-1997.patch

```diff
@@ -0,0 +1,25 @@
+From 735b7c82a16ddf16f39643830acb2ee62d2f3150 Mon Sep 17 00:00:00 2001
```

Comment:
I think this patch should live in `tools-local/patches` or something like that--to clarify what kind of patch this is and keep repo patches separate.

Reply:
Yes.  I moved it.